script/preview.js

```diff
@@ -53,62 +59,120 @@ if (options.buildpath === null) {
 const app = express();
 const drupalClient = getDrupalClient(options);
 
-app.use(express.static(path.join(__dirname, '..', options.buildpath)));
+const urls = {
+  [ENVIRONMENTS.LOCALHOST]: 'http://localhost:3001',
+  [ENVIRONMENTS.VAGOVDEV]:
+    'http://dev.va.gov.s3-website-us-gov-west-1.amazonaws.com',
+  [ENVIRONMENTS.VAGOVSTAGING]:
+    'http://staging.va.gov.s3-website-us-gov-west-1.amazonaws.com',
+  [ENVIRONMENTS.VAGOVPROD]:
+    'http://www.va.gov.s3-website-us-gov-west-1.amazonaws.com',
+};
```

Comment:
We have two different s3 buckets for each environment. These ones are what our *.va.gov environments point to and the others are asset buckets, which we're meant to reference assets from. They probably do make sense in that file, but I'll do that in a separate ticket, since I want to get this merged.